model_hub/model_hub/utils.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from typing import Callable, List
+
+import numpy as np
+from torch import Tensor
+
+from determined.pytorch import MetricReducer
+
+
+def configure_logging(level=logging.INFO):  # type: ignore
+    logging.basicConfig(
+        format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
+        datefmt="%m/%d/%Y %H:%M:%S",
+        level=level,
+    )
+
+
+def expand_like(arrays: List[np.ndarray], fill: float = -100) -> np.ndarray:
```

Comment:
it stacks arrays with potentially different sizes in dimension 1.  
if `dim A = (2, 4)` and `dim B = (2, 6)` it will create an array with dimension (4, 6) and fill in values in the first two rows by column 4 with the fill value.

Reply:
Sorry, forgot to mention this before.  Can we call this something like `vstack_ragged()`?  I think the name `expand_like()` does not reflect what it is doing very clearly.

Reply:
This is named expand_like because it offers the same functionality (with a different implementation) as a similarly named function in huggingface.